src/gen_caml.ml

```diff
@@ -290,8 +320,13 @@ let generate_stmt style index stmt =
     output "    in loop str";
     output "  in";
     output "  let sql = %s in" sql;
-    List.iter begin fun var ->
-      output "  let sql = replace_all ~str:sql ~sub:(\"%%%%%s%%%%\") ~by:%s in" var var;
+    List.iter begin function
+      | `Var var ->
+        output "  let sql = replace_all ~str:sql ~sub:(\"%%%%%s%%%%\") ~by:%s in" var var;
```

Comment:
we should use same pattern and unify these two cases one day (but that day is not today)